### PR TITLE
added notes about contribution/issue reporting

### PR DESCRIPTION
**Pull request by clxmstaab**
refs https://github.com/migrify/migrify/issues/123

I tried to make this as generic as possible, so it can be copy/paste'd into the other components.

### CODE REVIEW

File: packages/vendor-patches/README.md [hunk 1]
```diff
@@ -83,3 +83,11 @@ If not, get more information from composer to find out why:
...
 composer install --verbose
 ```
+
+## Report issues
+
+In case you are experiencing a bug or want to request a new feature head over to the [migrify monorepo issue tracker](https://github.com/migrify/migrify/issues)
+
+## Contribute
+
+The [sources of this package](https://github.com/migrify/migrify/tree/master/packages) are contained in the [migrify monorepo](https://github.com/migrify/migrify). We welcome contributions for this package on [migrify/migrify](https://github.com/migrify/migrify).
```
Comment (TomasVotruba) [inline packages/vendor-patches/README.md:93]: If I were a new user, I'm not sure which of these 3 links I should click.
Could you narrow it to 1 link so it's fool proof?

File: packages/vendor-patches/README.md [hunk 2]
```diff
@@ -83,3 +83,11 @@ If not, get more information from composer to find out why:
 ```bash
 composer install --verbose
 ```
+
+## Report issues
```
Comment (TomasVotruba) [inline packages/vendor-patches/README.md:87]:
```suggestion
## Report Issues
```

**Comment by clxmstaab** [comment]
done

**Comment by TomasVotruba** [comment]
Great! 

What's the plan now?

**Comment by clxmstaab** [comment]
just copied the text all over the repo components

**Comment by TomasVotruba** [comment]
Good. Could you squash it to 1 commit? I'll merge it then

**Comment by clxmstaab** [comment]
> Good. Could you squash it to 1 commit? I'll merge it then

couldn't you just hit the 'squash and merge' button on github?

**Comment by TomasVotruba** [comment]
Unfotunately, that doesn't squash commits here. So history of master and PR will be differnt.
In case of `git blame` on Github most commit relate to PR links like this one, so here is where we look for a regression. Having 5 commits instead of 1 for simple change makes it much harder. 

Morever if there is [~70 PRs per month merged](https://github.com/migrify/migrify/pulse/monthly).

**Comment by clxmstaab** [comment]
commits squashed

**Comment by TomasVotruba** [comment]
`master` yes, agreed. But this PR would contain 5 commits

**Comment by clxmstaab** [comment]
> `master` yes, agreed. But this PR would contain 5 commits.

why does it matter what this PR contains for the future?

**Comment by TomasVotruba** [comment]
That's what I tried to explain in 2nd paragraph here: https://github.com/migrify/migrify/pull/124#issuecomment-665614882

Is there a way I could write that better?

**Comment by TomasVotruba** [comment]
The idea is there is no added value for having 10 commits instead of 1 in pull-requests. Only the extra maintenance in reading.

**Pull request merged**